client/src/styles/Home.styles.ts

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PartialTheme, Theme } from '@fluentui/theme';
+import { CSSProperties } from 'react';
+
+export const fullScreenStyles = {
+  root: {
+    width: '100%',
+    height: '100%'
+  }
+};
+
+export function containerStyles(theme: PartialTheme | Theme | undefined): any {
+  return {
+    root: {
+      maxWidth: '64rem',
+      width: '100%',
+      height: '100%',
+      display: 'flex',
+      justifyContent: 'left', //use vertical and horizontal align instead
```

Comment:
I can see that `containerStyles` uses in `<Stack styles={containerStyles(props.theme)} tokens={{ childrenGap: 15 }}>` component and I believe that the `<Stack>` component has `horizontalAlign` and `verticalAlign`. I think it is better to use that instead of we manually handles it in styles to prevent potential confusion and keep consistency of maximizing Fluent UI.

What do you think about this?